### PR TITLE
update icon: rabbitmq (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by luizcarloscf**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

* Unify paths
* Removing "stroke" attributes
* Optimize icons

**This PR closes NONE**

## Notes

https://www.rabbitmq.com/
https://www.rabbitmq.com/trademark-guidelines.html

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Can you remove the padding for all icons so they fill the entire viewbox? Notice how there is some whitespace/padding around the icons here, which is not ideal for small sizes
image

**Comment by luizcarloscf** [comment]
Hi, @Snailedlt 

Done :rocket: :heavy_check_mark:

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this great contribution! 🚀

File: icons/rabbitmq/rabbitmq-plain-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2 0-3.252-5.41 0-4.488 2.155-6.11 3.633-6.11 1.487 0 3.16 1.53 3.16 5.89zM47.452 54.12v14.195c0 .768.025 5.9 4.738 5.9 1.137 0 5.276-.402 5.276-6.98 0-6.587-3.634-7.28-4.728-7.28a3.928 3.928 0 0 0-3.799 2.882h-.047V54.12h-1.449zm8.292 13.075c0 5.52-3.076 5.632-3.553 5.632-.477 0-3.248 0-3.248-5.41 0-4.488 2.155-6.11 3.637-6.11 1.479 0 3.164 1.53 3.164 5.89zm-19.605-3.67h1.75c.158-.795.5-2.214 2.834-2.214 3.053 0 2.95 2.657 2.95 3.68v1.198s-.994 0-1.61.06c-2.334.182-6.142.79-6.142 4.313 0 3.529 2.917 3.648 3.482 3.648a4.973 4.973 0 0 0 2.404-.576 5.05 5.05 0 0 0 1.821-1.666h.06V73.9h1.419v-9.313c0-1.395.127-4.643-4.253-4.643s-4.715 2.934-4.715 3.58zm7.53 4.03c0 .526-.037 1.055-.1 1.58a3.94 3.94 0 0 1-3.908 3.35 1.808 1.808 0 0 1-1.419-.477 1.843 1.843 0 0 1-.6-1.37c0-.725.278-2.415 3.049-2.947.585-.111 2.978-.134 2.978-.134zM23.217 54.122v19.78h1.709v-8.944h2.369a5.111 5.111 0 0 1 1.84.303c1.746.662 1.94 2.69 2.258 3.985l.99 4.68h1.72l-1.116-5.335c-.326-1.61-.549-2.298-.993-2.993-.593-.922-2.283-1.424-2.283-1.424 2.772-.54 3.976-2.408 3.976-4.95 0-2.336-1.126-5.062-6.308-5.062h-4.162zm1.709 1.574h2.258c4.51 0 4.777 2.404 4.777 3.494 0 .899-.171 4.194-5.473 4.194h-1.562zm0 0"/></svg>
```
Comment (Snailedlt) [inline icons/rabbitmq/rabbitmq-plain-wordmark.svg:1]: Add missing newline
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M116.432 72.557c1.824-1.566 2.802-4.488 2.802-8.701 0-6.138-2.313-10.077-6.63-10.077-4.047 0-6.87 3.299-6.87 10.077 0 6.777 2.398 10.056 6.75 10.056h8.142V72.25zm-8.822-8.697c0-5.637 1.607-8.372 4.974-8.372 3.363 0 4.754 3.272 4.754 8.372 0 5.104-1.224 8.39-4.839 8.39-3.616 0-4.889-2.754-4.889-8.39zM95.913 73.9h-1.797l-5.919-18.193V73.9H86.62V54.125h2.835l5.508 17.31h.06l5.529-17.31h2.762V73.9h-1.557V55.707zm29.198-3.032h-2.763v.242h1.255v2.835h.243V71.11h1.265zm2.626.242-.901 2.835h-.279l-.91-2.835v2.835h-.243v-3.077h.442l.863 2.699.858-2.7H128v3.078h-.246V71.11zM19.051 61.96h-6.348a.797.797 0 0 1-.8-.8v-6.363a.79.79 0 0 0-.794-.796h-2.38a.81.81 0 0 0-.563.23.804.804 0 0 0-.231.565v6.364a.82.82 0 0 1-.23.57.81.81 0 0 1-.565.23H4.764a.804.804 0 0 1-.564-.23.803.803 0 0 1-.23-.57v-6.363a.824.824 0 0 0-.235-.57.797.797 0 0 0-.566-.23H.794a.79.79 0 0 0-.794.799v18.31c0 .215.082.418.234.566a.768.768 0 0 0 .565.233h18.252a.797.797 0 0 0 .795-.8v-10.33a.794.794 0 0 0-.795-.815zm-3.192 6.76a1.185 1.185 0 0 1-1.189 1.198h-1.593a1.2 1.2 0 0 1-.843-.346 1.223 1.223 0 0 1-.346-.85v-1.587c0-.323.128-.624.346-.851a1.2 1.2 0 0 1 .843-.35h1.593a1.193 1.193 0 0 1 1.19 1.2zm67.624 5.37c-.477.065-.962.109-1.45.124a4.216 4.216 0 0 1-2.708-.783c-.596-.5-.894-1.375-.894-2.575V61.62h-2.174v-1.363h2.174v-3.153l1.415-.955v4.1h3.637v1.366h-3.63v8.778c0 .393-.094 2.428 2.06 2.428a5.4 5.4 0 0 0 1.57-.246zm-9.616-19.966v2.318h-2.11v-2.317zm-.33 6.134V73.9H72.11V60.258zM59.681 54.12v14.195c0 .768.023 5.9 4.742 5.9 1.133 0 5.267-.402 5.267-6.98 0-6.587-3.63-7.28-4.722-7.28a3.928 3.928 0 0 0-3.8 2.882h-.061V54.12h-1.442zm8.288 13.075c0 5.52-3.073 5.632-3.546 5.632-.482 0-3.252 0-3.252-5.41 0-4.488 2.155-6.11 3.633-6.11 1.487 0 3.16 1.53 3.16 5.89zM47.452 54.12v14.195c0 .768.025 5.9 4.738 5.9 1.137 0 5.276-.402 5.276-6.98 0-6.587-3.634-7.28-4.728-7.28a3.928 3.928 0 0 0-3.799 2.882h-.047V54.12h-1.449zm8.292 13.075c0 5.52-3.076 5.632-3.553 5.632-.477 0-3.248 0-3.248-5.41 0-4.488 2.155-6.11 3.637-6.11 1.479 0 3.164 1.53 3.164 5.89zm-19.605-3.67h1.75c.158-.795.5-2.214 2.834-2.214 3.053 0 2.95 2.657 2.95 3.68v1.198s-.994 0-1.61.06c-2.334.182-6.142.79-6.142 4.313 0 3.529 2.917 3.648 3.482 3.648a4.973 4.973 0 0 0 2.404-.576 5.05 5.05 0 0 0 1.821-1.666h.06V73.9h1.419v-9.313c0-1.395.127-4.643-4.253-4.643s-4.715 2.934-4.715 3.58zm7.53 4.03c0 .526-.037 1.055-.1 1.58a3.94 3.94 0 0 1-3.908 3.35 1.808 1.808 0 0 1-1.419-.477 1.843 1.843 0 0 1-.6-1.37c0-.725.278-2.415 3.049-2.947.585-.111 2.978-.134 2.978-.134zM23.217 54.122v19.78h1.709v-8.944h2.369a5.111 5.111 0 0 1 1.84.303c1.746.662 1.94 2.69 2.258 3.985l.99 4.68h1.72l-1.116-5.335c-.326-1.61-.549-2.298-.993-2.993-.593-.922-2.283-1.424-2.283-1.424 2.772-.54 3.976-2.408 3.976-4.95 0-2.336-1.126-5.062-6.308-5.062h-4.162zm1.709 1.574h2.258c4.51 0 4.777 2.404 4.777 3.494 0 .899-.171 4.194-5.473 4.194h-1.562zm0 0"/></svg>

```

**Review by Snailedlt — APPROVED** [review]
✔️ 
Thanks again! 🙇

**Pull request merged**